source/products/foyer/foyer-young-driver/webapp.rst

```diff
@@ -0,0 +1,41 @@
+Webapp
+======
+
+Example Kasko JS embed code
+---------------------------
+
+.. code:: html
+
+    <script type="text/javascript" src="https://eu1.kaskojs.com/v2"></script>
+    <div id="kasko-widget-container"></div>
+    <script>
```

Comment:
Should I add the snippet that we use in wix? 🤔

Reply:
@tomstumshais

Reply:
Where you want to use it? Also in Wix?

Reply:
yes its used in wix (it has all the necessary functions) i thought maybe should add here just in case 🤷‍♀️

Reply:
Good point. In that way we can have backup for Wix code 👍